lib/extend/console.js

```diff
@@ -5,17 +5,21 @@ const abbrev = require('abbrev');
 
 class Console {
   constructor() {
-    this.store = {};
+    this.store = new Map();
     this.alias = {};
   }
 
   get(name) {
     name = name.toLowerCase();
-    return this.store[this.alias[name]];
+    return this.store.get(this.alias[name]);
   }
 
   list() {
-    return this.store;
+    const result = {};
+    for (const [key, value] of this.store) {
+      result[key] = value;
+    }
+    return result;
```

Comment:
```suggestion
    // FIXME: Could use Object.fromEntries after Node.js 10 is dropped.
    const result = {};
    for (const [key, value] of this.store) {
      result[key] = value;
    }
    return result;
```

Could add a `FIXME` message.